frappe/public/js/frappe/ui/link_preview.js

```diff
@@ -190,7 +190,7 @@ frappe.ui.LinkPreview = class {
 		$popover.addClass('link-preview-popover');
 		$popover.toggleClass('control-field-popover', this.is_link);
 
-		this.$links.push(this.element);
+		this.popovers.push(this.element.data('bs.popover'));
```

Comment:
should be `this.popover` ?

Reply:
`popovers` because it's a list of popovers. I'll rename it to `this.popovers_list`?